scripts/verify_genproto.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# This scripts is automatically run by CI to prevent pull requests missing running genproto.sh
+# after changing *.proto file.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+./scripts/genproto.sh
+diff=$(git diff --name-only | grep -c ".pb.")
```

Comment:
This implementation checks if there is any uncommitted proto files, this should work for CI, but not for local development. 
I think this should be ok, but maybe you can figure out if we could output `./scripts/genproto.sh` to tmp directory and compare it's contents to local files.